client/src/saga/signInSaga.js

```diff
@@ -0,0 +1,26 @@
+import { takeEvery, put } from 'redux-saga/effects'
+import { DO_SIGN_IN_SAGA } from '../redux/constants'
+import { addUserToStore } from '../redux/actions/actionUser'
+
+function* doSignInSaga({ data }) {
+  try {
+    const response = yield fetch('/auth/register',
+      {
+        method: 'POST',
+        body: JSON.stringify(data),
+        headers: { 'Content-Type': 'application/json' },
+      })
```

Comment:
The same parameters are very likely to be applied to any request to the server. Let's create `post` and `get` utils sometime later in a separate PR as described here https://trello.com/c/LDfBFkEN/11-create-post-and-get-helper-utils

Reply:
I'll do it in the future - when I'll do requests to DB